login/service.go

```diff
@@ -1091,3 +1092,42 @@ func TokenToJson(ctx context.Context, outhToken *oauth2.Token) (string, error) {
 
 	return string(b), nil
 }
+
+// UserInfo gets user infomation given a context containing access_token
+func (keycloak *KeycloakOAuthProvider) UserInfo(ctx context.Context) (*account.User, *account.Identity, error) {
```

Comment:
not needed anymore.

Reply:
https://github.com/fabric8-services/fabric8-auth/pull/313/commits/7c46e498276b11657b352032576054621093f36a